primitives/src/lib.rs

```diff
@@ -16,6 +16,7 @@ mod app {
     use sp_application_crypto::{app_crypto, ed25519};
     app_crypto!(ed25519, crate::KEY_TYPE);
 }
+pub use app::Public;
```

Comment:
Huh? Where is this used?

Reply:
`     authorities: Vec<aleph_primitives::Public>,` in `finality-aleph/src/party/mod.rs`